src/test/kotlin/BlackjackGameTest.kt

```diff
@@ -0,0 +1,248 @@
+import model.Card
+import model.CardNumber
+import model.CardShape
+import model.CardVendor
+import model.Deck
+import model.Names
+import model.Players
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertAll
+
+internal class BlackjackGameTest {
+    @Test
+    fun `카드 번호가 ACE 인지 확인한다`() {
+        assertAll(
+            {
+                assertThat(CardNumber.ACE.isAce()).isTrue
+            },
+            {
+                assertThat(CardNumber.KING.isAce()).isFalse
+            }
+        )
+    }
+
+    @Test
+    fun `카드 번호가 문자열로 변경된다`() {
+        assertAll(
+            {
+                assertThat(CardNumber.ACE.toString()).isEqualTo("A")
+            },
+            {
+                assertThat(CardNumber.KING.toString()).isEqualTo("K")
+            },
+            {
+                assertThat(CardNumber.TWO.toString()).isEqualTo("2")
+            }
+        )
+    }
+
+    @Test
+    fun `카드 모양이 문자열로 변경된다`() {
+        assertAll(
+            {
+                assertThat(CardShape.DIAMONDS.toString()).isEqualTo("다이아몬드")
+            },
+            {
+                assertThat(CardShape.CLUBS.toString()).isEqualTo("클로버")
+            }
+        )
+    }
+
+    @Test
+    fun `카드 숫자 합이 21 미만일 경우 추가 카드 받기가 가능하다`() {
+        assertAll(
+            {
+                assertThat(
+                    CardVendor().isGetExtraCard(
+                        listOf(
+                            Card(
+                                CardNumber.FIVE,
+                                CardShape.SPADES
+                            )
+                        )
+                    )
+                ).isTrue()
+            },
+            {
+                assertThat(
+                    CardVendor().isGetExtraCard(
+                        listOf(
+                            Card(CardNumber.TEN, CardShape.SPADES),
+                            Card(CardNumber.KING, CardShape.HEARTS),
+                            Card(CardNumber.ACE, CardShape.CLUBS)
+                        )
+                    )
+                ).isFalse()
+            }
+        )
+    }
+
+    @Test
+    fun `나머지 카드 숫자 합이 0이상 10이하 일 경우 ACE 카드 값을 11로 결정한다`() {
+        assertAll(
+            {
+                assertThat(CardVendor().decideAceCardNumber(2)).isEqualTo(11)
+            },
+            {
+                assertThat(CardVendor().decideAceCardNumber(10)).isEqualTo(11)
+            }
+        )
+    }
+
+    @Test
+    fun `나머지 카드 숫자 합이 0이상 10이하가 아닌 경우 ACE 카드 값을 1로 결정한다`() {
```

Comment:
```suggestion
    fun `나머지 카드 숫자 합이 11이상일 경우 ACE 카드 값을 1로 결정한다`() {
```
테스트 코드를 보는 사람 입장에서는 0이상 10이하가 아닌 경우라고 한다면, 어떤 경우인지 생각을 하게 됩니다.
굳이 생각하게 할 필요없이 명시적으로 작성해보면 어떨까요? (이하 관련 내용 동일)